meilisearch-release-changelog/v1.1.0.md

```diff
@@ -0,0 +1,178 @@
+    {
+      "indexUid": "products",
+      "q": "Nike",
+      "limit": 1
+    },
+    {
+      "indexUid": "brands",
+      "q": "Nike",
+      "limit": 1
+    }
+  ]
+}
+```
+
+In this case, Meilisearch returns an array of results for each index:
+
+```json
+{
+	"results": [
+	  {
+	    "indexUid": "products",
+	    "hits": [...],
+	    "query": "Nike",
+	    "processingTimeMs": 1,
+	    "limit": 1,
+	    "offset": 0,
+	    "estimatedTotalHits": 17
+	  },
+	  {
+	    "indexUid": "brands",
+	    "hits": [...],
+	    "query": "Nike",
+	    "processingTimeMs": 0,
+	    "limit": 1,
+	    "offset": 0,
+	    "estimatedTotalHits": 7
+	  }
+	]
+}
+```
+
+Done by @dureuill in #3417
+
+## Introduce facet stats
+
+When using the `facets` parameter during a search, statistics about the distributed facets are displayed in a `facetStats` object. This object will contain the minimum (`min`) and the maximum (`max`) of each numerical facet corersponding to search query.
+
+Nothing needs to be activated, this information are available by default when requiring facets during a search (with the `facets` parameter).
+
+Done by @dureuill in #3423.
+
+## Geo-search improvements: introcdution of `geoBoundingBox`
+
+When doing a filter (with the `filter` parameter) during the search, you can now use the `geoBoundingBox`: `_geoBoundingBox([lat, lng], [lat, lng])`, where the first argument is the top left corner, and the second one the bottom right corner.
+
+Done by @gmourier and @irevoire in #3405 and #3529.
+
+## Lifting the limits on index size and count
+
+Before v1.0, the maximum size of an index on disk was limited according to the `--max-index-size` CLI parameter, and the number of indexes that could be accessed by one Meilisearch process would depend on this value (typically, about 1000 indexes with the default setting of 100GiB for `--max-index-size`).
+
+v1.0 removed `--max-index-size` and fixed the maximum size of indexes to 500GiB, with about 200 concurrent indexes accessible to a Meilisearch process.
+
+The removal of `--max-index-size` was made in preparation for this release: you can now use an unlimited number of indexes, whose maximum size will be only dictated by the memory address space devoted to a single process by the OS. The limit is at about 80TiB under Linux, you might run out of disk space before reaching it!
+
+Done by @dureuill in #3319, #3331 and #3544.
+
+## Experimental feature: monitore the Meilisearch usage by using Prometheus
+
+This release introduces an experimental feature to follow your metrics by using Prometheus.
+
+When launching Meilisearch, use the following flag:
+
+```bash
+meilisearch --experimental-enable-metrics
+```
+
+The `/metrics` route will be available and compatible with Prometheus.
+
+🗣️ Since this feature is experimental, we need you to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/meilisearch/discussions/3518).
```

Comment:
```suggestion
🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/meilisearch/discussions/3518).
```